source/help/apps/desktop-changelog.rst

```diff
@@ -31,7 +31,7 @@ Bug Fixes
 
 All Platforms
 ^^^^^^^^^^^^^
-- Fixed an issue where the app window started as maximized when "Start app on login" setting was enabled. This setting is not respected when AppImage file (Unofficial) is used.
+- Fixed an issue where the app window started as maximized when "Start app on login" setting was enabled. The Desktop App no longer starts to system tray and the parameter ``--hidden`` was removed. This setting is not respected when AppImage file (Unofficial) is used.
```

Comment:
```suggestion
- Fixed an issue where the app window started as maximized when the "Start app on login" setting was enabled. The Desktop App no longer starts to system tray and the parameter ``--hidden`` was removed. This setting is not respected when AppImage file (Unofficial) is used.
```

Reply:
I'm not clear on what "starts to system tray" means in this context.

Reply:
it was a parameter that confused people. starts to system tray means it will start but no window will appear, only a small icon in the lower-right corner of windows/linux which could be used to restore the window.

Reply:
Clarified the text for this,